go.mod

```diff
@@ -21,16 +21,20 @@ require (
 	cosmossdk.io/log v1.4.1
 	cosmossdk.io/store v1.1.1-0.20240418092142-896cdf1971bc
 	cosmossdk.io/store/v2 v2.0.0-20240821144902-e88c138760a3
+	github.com/bazelbuild/buildtools v0.0.0-20241129155226-a0444eb13952
```

Comment:
@fridrik01 how do you explain these changes?

Reply:
Going by memory as I don't have my main dev machine with me. 

I think I may have updated the buildtools manually to get the newest version as the `buildifier-install` make target was failing. I had thought we could remove this dependency (I think I did in the first few commits) but then I observed that many of our linters (buildifier-install, golines.sh, buf-install) relied on go install which was not working for me locally. Therefore I pulled the dependencies back into `build/tools/types.go` but we should aim for putting these dependencies outside of the main go.som/go.mod files as they bring in IIRC ~200 extra indirect dependencies.